src/test/java/roomescape/MissionStepTest.java

```diff
@@ -18,10 +20,100 @@
 public class MissionStepTest {
 
     @Test
-    void 일단계() {
+    @DisplayName("로그인 요청에 대한 응답 API 테스트")
```

Comment:
```suggestion
    @DisplayName("로그인 요청 시 쿠키에서 토큰을 가져온다.")
```

 테스트 코드의 목적이 이미 명확하기 때문에 테스트임을 명시할 필요는 없을 것 같아요!

Reply:
감사합니다! 수정했습니당